message_ix_workshop.rst

```diff
@@ -2,53 +2,111 @@ MESSAGEix Workshop
 ^^^^^^^^^^^^^^^^^^
 
 The aim of the workshop is to help new users of the MESSAGEix
-modelling framework to get started with their modeling work.
+modelling framework getting started with their modelling work.
 The main features of the “framework” are introduced, and the
 use cases of some features are shown. The user can learn how
-to build an energy model and how to represent some policy constraints
-in their energy scenarios.
+to build an energy model and how to represent some policy
+constraints in their energy scenarios.
 For information about the model, its structure, mathematical
-formulation and much more, please see the documentation
-at: https://docs.messageix.org.
-The different lectures contain the workshop slides, videos as
-well as tutorials for hands-on examples.
+formulation and much more, please see the `documentation <https://docs.messageix.org>`_.
 
-Introduction to MESSAGEix Modelling Framework
-"""""""""""""""""""""""""""""""""""""""""""""
-Introduction to the MESSAGEix modeling framework and its components.
+.. contents:: Table of Content
+   :local:
 
+Recent workshops
+""""""""""""""""
 
-Getting started with the framework.
+Under the following links you will find the slides as
+well as the recordings of the last workshops:
 
+`MESSAGEix-Workshop material (June 2021) <http://pure.iiasa.ac.at/id/eprint/17318/>`_
```

Comment:
I would flip this around, focus on the date, then have the link to the material in brackets - and add one or two previous workshops (without putting the material on pure), just to highlight that we are doing this regularly.

Reply:
How about leaving out the "MESSAGEix workshop materials" altogether and just giving the dates with the links, since it is obvious that these are the workshop materials and it would be redundant to have multiple "materials". Just like that:

- xx 2020
- xx 2020
- June 2021

Reply:
Of course, the link to the materials should be shown only once. No preference whether this is included next to the most recent workshop or as a separate line before/after the list of workshops.

In any case, I would flip the order and show the most recent workshop first.